spec/models/king_spec.rb

```diff
@@ -25,4 +25,46 @@
       expect(king.valid_move?(Coordinates.new(0, 4))).to eq false
     end
   end
+
+  describe '#can_castle' do
+    let(:game) { create(:game) }
+    let(:king) { game.chess_pieces.find_by(position_x: 5, position_y: 8) }
+    let(:rook) { game.chess_pieces.find_by(position_x: 1, position_y: 8) }
+
+    it 'returns false if a rook has previously moved' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.